java/flight/flight-sql/src/test/java/org/apache/arrow/flight/TestFlightSql.java

```diff
@@ -67,12 +67,12 @@
 import org.apache.arrow.vector.types.pojo.Schema;
 import org.apache.arrow.vector.util.Text;
 import org.hamcrest.Matcher;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ErrorCollector;
+import org.hamcrest.MatcherAssert;
```

Comment:
Note there's a merge conflict on this file now

Reply:
@lidavidm Fixed